core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/toDataFrame.kt

```diff
@@ -38,7 +38,7 @@ public inline fun <reified T> Iterable<T>.toDataFrame(vararg props: KProperty<*>
         properties(roots = props, maxDepth = maxDepth)
     }
 
-public inline fun <reified T> DataColumn<T>.read(): AnyCol = when (kind()) {
+public inline fun <reified T> DataColumn<T>.unfold(): AnyCol = when (kind()) {
     ColumnKind.Group, ColumnKind.Frame -> this
     else -> when {
         isPrimitive() -> this
```

Comment:
While the name `unfold()` does make sense for `isPrimitive()` and the `else` statement where the object is turned into a dataframe one property at a time, for the reading  of a file or url the name `read()` makes way more sense. Would it be a good idea to have both names? Or is this what you're planning to do?

Reply:
I reckon people would use it for the same purpose, thus one function and one name.
It's not clear what function does from its name though. Even `read` is more like `read` + `replace`. So people would have to read docs / kdocs to find out anyway.

Reply:
speaking of, how would you make kdocs in this situation? i'm in dither over this because duplication and maintenance

Reply:
hmm I'd definitely keep the old functions, annotate them with 
```kotlin
@Deprecated("read is now called unfold", ReplaceWith("unfold"), DeprecationLevel.ERROR)
```
And then simply KDoc the `read()` functions with `@see unfold`

Reply:
But what does read/unfold do with a file or url? I'd say it still "read"s those right? I'd see "unfold" as only "unfolding" an object into a dataframe.